scripts/xcm_transfers/xcm/registry/reserve_location.py

```diff
@@ -0,0 +1,89 @@
+from typing import Tuple, Callable, List
```

Comment:
There is no usage for List
```suggestion
from typing import Tuple, Callable
```